plugins/nanospell/plugin.js

```diff
@@ -887,12 +885,6 @@
 			range.extractContents().appendTo(span);
 			range.insertNode(span);
 		},
-		markTypos: function (editor, node) {
-			var range = editor.createRange();
-			range.selectNodeContents(node);
-
-			this.markTyposInRange(editor, range);
-		},
```

Comment:
I think I'd rather keep this method and then have scanWords call it instead.  Mostly for clarity reasons, and also, I suspect that in the future, we may have some cases where we'll want to be smarter about doing element checking vs. actual range checking.  (For example, copy/paste or drag and drop, which don't happen as often, but we haven't really thought about at all)